.github/workflows/docs.yml

```
@@ -0,0 +1,61 @@
+name: Docs CI
+
+on:
+  push:
+    tags:
+      - "*"
+  pull_request:
+
+jobs:
+  docs:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["ubuntu-latest"]
+        python: ["3.10"]
+
+    runs-on: ubuntu-latest
```

Either 
```suggestion
    runs-on: ${{ matrix.os }}
```
or delete matrix.os

---

deleted os